integrations/malware_tools_analyzers/clamav/freshclam.conf

```diff
@@ -15,5 +15,4 @@ Foreground true
 DatabaseMirror db.local.clamav.net
 DatabaseMirror database.clamav.net
 ScriptedUpdates yes
-NotifyClamd /etc/clamav/clamd.conf
```

Comment:
Was this an intentional change or got included by mistake ?

Reply:
I messed up my git rebase and included commit e037d32f44bc6c531e1eae9e6917142ff36ac03b in it

Reply:
yes that was mine